src/components/SponsorCard/SponsorCard.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import SponsorCard from './SponsorCard';
+
+describe('SponsorCard', () => {
+  test('renders SponsorCard component', () => {
+    render(<SponsorCard value="Hello" />);
+
+    expect(screen.getByText('Hello'));
+  });
+});
```

Comment:
here we have error inside test suite. instead value we need props from Sponsor card and some test to mach those values, for ex if src exist or its name